chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -231,7 +231,15 @@
   action :create
 end
 
-unless node["crowbar"].nil? or node["crowbar"]["users"].nil? or node["crowbar"]["realm"].nil?
+if !node["crowbar"].nil? && !node["crowbar"]["realm"].nil?
```

Comment:
Could this be written as
`if node["crowbar"] && node["crowbar"]["realm"]` ?

Reply:
I guess it can, yes. I didn't want to move away from the `unless` syntax to keep changes minimal but hound made me do this ;-)

Reply:
Hm, I don't quite get that reasoning :). The current state is rather ugly.